### ISSUE TITLE
No OperatorHub items found

### ISSUE THREAD

**Issue by yselkowitz** [openshift/ocp-on-arm#1]
As part of cluster bringup, the operator-marketplace usually installs [four CatalogSource operator indexes](https://github.com/operator-framework/operator-marketplace/tree/master/defaults) which contain the data to populate the OperatorHub in the admin console.   At this point in 4.9 development, these still point to the 4.8 indexes which do not include ARM, and the 4.9 operators have yet to be made publicly available.  Therefore, the defaults are disabled to avoid unnecessary alerts, and the OperatorHub is empty.

The `[sig-operator] an end user can use OLM can subscribe to the operator [Suite:openshift/conformance/parallel]` test may also fail for this reason, but is not indicative of a functional issue in OperatorHub.

**Comment by yselkowitz** [comment]
This was just fixed in the development branch for 4.10, and is in the process of being backported to 4.9.  In the meantime, it is now possible to install the default indexes manually:
```
oc create -f https://raw.githubusercontent.com/operator-framework/operator-marketplace/release-4.9/defaults/01_redhat_operators.cr.yaml
oc create -f https://raw.githubusercontent.com/operator-framework/operator-marketplace/release-4.9/defaults/02_certified_operators.yaml
oc create -f https://raw.githubusercontent.com/operator-framework/operator-marketplace/release-4.9/defaults/03_community_operators.yaml
oc create -f https://raw.githubusercontent.com/operator-framework/operator-marketplace/release-4.9/defaults/04_redhat_marketplace.yaml
```
After a few moments, a list of operators should be displayed on the OperatorHub, from where you can install them as usual.  Please note that only a limited selection of operators are currently available for ARM.

/remove-lifecycle

**Comment by yselkowitz** [comment]
The 4.9.10 release, now available in the candidate-4.9 channel, includes a fully functional OperatorHub out of the box.

**Comment by yselkowitz** [comment]
4.9.10 is now available in the fast-4.9 channel.

**Comment by yselkowitz** [comment]
4.9.10 is now available in the stable-4.9 channel, so all users should be upgraded to this shortly, and new installs will no longer see this.
